src/app/components/feed/FeedMediaRequests.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable relay/unused-fields */
```

Comment:
Why are we disabling it here? Adding a comment could guide future refactoring out of it.

Reply:
Thanks, I added a comment with this explanation: We're using the legacy `<Annotations />` component, which doesn't define a fragment.